cmd/openqa-revtui/openqa-revtui.go

```diff
@@ -598,6 +599,17 @@ func tui_main(tui *TUI, instance gopenqa.Instance) error {
 			// Shift through the sorting mechanism
 			tui.SetSorting((tui.Sorting() + 1) % 2)
 			tui.Update()
+		} else if key == 'o' {
+			for _, job := range tui.Model.jobs {
+				if !tui.hideJob(job) {
+					var err error
+					err = exec.Command("xdg-open", job.Link).Start()
+					if err != nil {
+						tui.SetStatus(fmt.Sprintf("Error: %s", err))
```

Comment:
```suggestion
						tui.SetStatus(fmt.Sprintf("Error: %s", err))
						tui.Update()
						return
```

In the error case it likely will fail for all subsequent calls as well, so it makes little sense to iterate the same error x times.

Reply:
generally agree but I have different idea how to achieve the same . Please check my version if you don't like it I will switch to yours

Reply:
Jep, a `break` also works nicely.